tchannel/retry.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+NEVER = 'n'
+CONNECTION_ERROR = 'c'
+TIMEOUT = 't'
+CONNECTION_ERROR_AND_TIMEOUT = 'ct'
+DEFAULT = CONNECTION_ERROR
+
```

Comment:
Docs on all.

Use the following format to get sphinx to pick it up.

    #: docstring
    CONSTANT = value